pages/views/_includes/footer.njk

```diff
@@ -25,7 +25,14 @@
     </ul>
   </esl-panel-group>
   <div class="footer-dropdown footer-links">
-    <a href="#" class="footer-link">Terms & Conditions</a>
-    <a href="#" class="footer-link">Privacy Policy</a>
+    <div class="footer-link">
+      <a href="#">Terms & Conditions</a>
+      <a href="#">Privacy Policy</a>
+    </div>
```

Comment:
something wrong here
- no classes on anchors
- single footer-link for two anchors